src/components/molecules/chat-space-elements/chat-space-body/ChatSpaceBody.tsx

```diff
@@ -47,48 +67,64 @@ const ChatSpaceBody = (props: {
 
         return (
           <div key={message._id}>
-            {isNewDate && <TimeStamp date={createdAtDate} />}
-            {isGuestMessage ? (
-              <S.ChatBubbleGuestContainer>
-                {chatPartners?.userImage && (
-                  <S.ChatGuestImage
-                    src={chatPartners.userImage}
-                    alt="GuestImage"
-                  />
-                )}
-                <S.ChatBubbleGuestCenter>
-                  <S.ChatGuestName>{chatPartners?.name}</S.ChatGuestName>
-                  <S.ChatBubble isHost={isGuestMessage}>
-                    <span>{message.content}</span>
-                  </S.ChatBubble>
-                </S.ChatBubbleGuestCenter>
-                {isValidTime && (
-                  <S.ChatTimeBox>{`${getAmPm(hours)} ${
-                    hours % 12 || 12
-                  }:${minutes}`}</S.ChatTimeBox>
-                )}
-              </S.ChatBubbleGuestContainer>
-            ) : (
-              <S.ChatBubbleHostContainer>
-                {!isSameDay(currentDate, createdAtDate) && (
-                  <TimeStamp date={createdAtDate} />
-                )}
-                {/* 일단 TimeStamp가 상대방이나 본인이 채팅했을 때 날짜가 넘어가면
+            <div
+              onContextMenu={handleChatDelete(message._id)}
+              data-message-id={message._id}>
```

Comment:
```suggestion
            <div
              onContextMenu={handleChatDelete(message._id)}
              data-message-id={message._id}>
```
제 생각엔 여기가 문제지 않을까 싶네요 map함수를 돌린 상태에서 message._id 를 넘겨주고 있으니까 전체의 id가 담기는게 아닐까..